Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/resultavailable/SubmissionTestResultAvailableFragment.kt

```diff
@@ -83,7 +83,7 @@ class SubmissionTestResultAvailableFragment : Fragment(R.layout.fragment_submiss
         var dialogTitle = R.string.submission_test_result_available_close_dialog_title_consent_not_given
         var dialogBody = R.string.submission_test_result_available_close_dialog_body_consent_not_given
 
-        if (binding.submissionTestResultAvailableConsentStatus.consent) {
+        if (!binding.submissionTestResultAvailableConsentStatus.consent) {
```

Comment:
I think your fix is valid, but if I read this:
1. consent not given strings are set
2. consent isn't given -> consent given strings are used

Are the strings maybe named wrong?

Reply:
Yes, it looks like they are. The last changes pushed should take care of this.